lib/plugins/package/tests/packageService.js

```diff
@@ -129,27 +167,32 @@ describe('#packageService()', () => {
     it('should call zipService with settings', () => {
       const servicePath = 'test';
       const exclude = ['test-exclude'];
+      const include = ['test-include'];
       const artifactName = 'test-artifact.zip';
       const artifactFilePath = '/some/fake/path/test-artifact.zip';
 
       serverless.config.servicePath = servicePath;
 
-      const getExcludedPathsStub = sinon
-        .stub(packageService, 'getExcludedPaths').returns(exclude);
+      const getExcludesStub = sinon
+        .stub(packageService, 'getExcludes').returns(exclude);
+      const getIncludesStub = sinon
+        .stub(packageService, 'getIncludes').returns(include);
       const getServiceArtifactNameStub = sinon
         .stub(packageService, 'getServiceArtifactName').returns(artifactName);
 
       const zipDirectoryStub = sinon
         .stub(packageService, 'zipDirectory').returns(BbPromise.resolve(artifactFilePath));
 
       return packageService.packageAll().then(() => {
-        expect(getExcludedPathsStub.calledOnce).to.be.equal(true);
+        expect(getExcludesStub.calledOnce).to.be.equal(true);
+        expect(getIncludesStub.calledOnce).to.be.equal(true);
         expect(getServiceArtifactNameStub.calledOnce).to.be.equal(true);
 
         expect(zipDirectoryStub.calledOnce).to.be.equal(true);
         expect(zipDirectoryStub.args[0][0]).to.be.equal(servicePath);
```

Comment:
Instead of matching arguments like this I personally find it way more expressive to use sinon's `calledWithExactly`
https://github.com/serverless/serverless/blob/master/lib/plugins/aws/deploy/tests/cleanupS3Bucket.js#L207